src/roiextractors/extractors/caiman/caimansegmentationextractor.py

```diff
@@ -81,12 +81,17 @@ def get_accepted_list(self):
         accepted = self._dataset_file["estimates"]["idx_components"]
         if len(accepted.shape) == 0:
             accepted = list(range(self.get_num_rois()))
+        else:
+            accepted = list(accepted[:])
         return accepted
 
     def get_rejected_list(self):
         rejected = self._dataset_file["estimates"]["idx_components_bad"]
-        if len(rejected.shape) > 0:
-            return rejected
+        if len(rejected.shape) == 0:
+            rejected = list()
+        else:
+            rejected = list(rejected[:])
+        return rejected
```

Comment:
So this returning `None` was the source of the original error?

Reply:
Yes, this and the one above. This one was a typing check if I remember well.